dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/BatchImport.java

```diff
@@ -74,54 +63,6 @@ public Response postImport(String body, @QueryParam("dv") String parentIdtf, @Qu
         }
     }
 
-    /**
-     * Import single or multiple datasets that are in the local filesystem
-     *
-     * @param fileDir    the absolute path of the file or directory (all files
-     *                   within the directory will be imported
-     * @param parentIdtf the dataverse to import into (id or alias)
-     * @param apiKey     user's api key
-     * @return import status (including id's of the datasets created)
-     */
-    @GET
-    @ApiWriteOperation
-    @Path("import")
-    public Response getImport(@QueryParam("path") String fileDir, @QueryParam("dv") String parentIdtf, @QueryParam("createDV") Boolean createDV, @QueryParam("key") String apiKey) {
-
-        return startBatchJob(fileDir, parentIdtf, apiKey, ImportType.NEW, createDV);
-
-    }
-
-    private Response startBatchJob(String fileDir, String parentIdtf, String apiKey, ImportType importType, Boolean createDV) {
-        if (createDV == null) {
-            createDV = Boolean.FALSE;
-        }
-        try {
-            DataverseRequest dataverseRequest;
-            try {
-                dataverseRequest = createDataverseRequest(findAuthenticatedUserOrDie());
-            } catch (WrappedResponse wr) {
-                return wr.getResponse();
-            }
-            if (parentIdtf == null) {
-                parentIdtf = "root";
-            }
-            Dataverse owner = findDataverse(parentIdtf);
-            if (owner == null) {
-                if (createDV) {
-                    owner = importService.createDataverse(parentIdtf, dataverseRequest);
-                } else {
-                    return error(Response.Status.NOT_FOUND, "Can't find dataverse with identifier='" + parentIdtf + "'");
-                }
-            }
-            batchService.processFilePath(fileDir, parentIdtf, dataverseRequest, owner, importType, createDV);
```

Comment:
Can we make deletions more deep? Probably `batchService.processFilePath` will be not used anymore. And it will probably spawn more methods that could be removed